### ISSUE TITLE
"opencart-2checkout-api" is not working with Opencart3 or Opencart 1.5.6.4

### ISSUE THREAD

**Issue by abdulwad7869** [craigchristenson/opencart-2checkout-api#18]
I have tried your plugin on my both websites but it is not working with either one. Kindly tell me how can I get this plugin work for me?

**Comment by condor2** [comment]
Becase wasn't made for 3.x or 1.5.

**Comment by abdulwad7869** [comment]
Is there any working 2checkout extension for opencart3?

**Comment by condor2** [comment]
You can hire somebody to convert.
![Screenshot_2019-11-08_17-13-31](https://user-images.githubusercontent.com/14251939/68487079-2254b180-024b-11ea-9108-432201c2d1a6.png)

**Comment by abdulwad7869** [comment]
Thanks. I can't afford to convert it. But I will try to convert it myself.
I have tried 2Checkout version from 3.x, it was always fetching price incorrectly like default currency is Euro, order placed in 200 euro, 2Checkout will show 200$ instead of charging in Euro. 
2nd client has to enter his/her data again on 2Checkout form. I want to design one-page checkout f. Only 2Checkout is available in Pakistan for payment receiving online. 
I am trying to solve these issues but not able to solve them. Also, it was not showing cart after redirection it created Invoice link for redirection without website header and footer.
I am looking for its solution but still no luck. If you have any information let me know. Also, give me hints how can I convert this extension for Opencart 3.x.

**Comment by abdulwad7869** [comment]
Can you tell exactly which version of Opencart work with this plugin?

**Comment by craigchristenson** [comment]
condor2 and I are working on a version for 3.x, you can checkout the branch at https://github.com/craigchristenson/opencart-2checkout-api/tree/condor2-v3.0

**Comment by abdulwad7869** [comment]
> condor2 and I are working on a version for 3.x, you can checkout the branch at https://github.com/craigchristenson/opencart-2checkout-api/tree/condor2-v3.0

I have tried the new version but it is giving "Unauthorized" error when card data is filled.

**Comment by craigchristenson** [comment]
That error comes from 2Checkout and means that your 2Checkout publishable and private key credentials are wrong. Make sure that you are using your production credentals and that you have sandbox mode unchecked in the configuration.

**Comment by abdulwad7869** [comment]
> That error comes from 2Checkout and means that your 2Checkout publishable and private key credentials are wrong. Make sure that you are using your production credentals and that you have sandbox mode unchecked in the configuration.

Thanks. I will try that and let you know.

**Comment by abdulwad7869** [comment]
> That error comes from 2Checkout and means that your 2Checkout publishable and private key credentials are wrong. Make sure that you are using your production credentals and that you have sandbox mode unchecked in the configuration.

I have tried your new extension but it is still not authorized to make payment hence all information is correct. I have checked the information several times with your instructions, 2checkout details, and opencart details. There was only "Secret Word" in opencart extension, I could not find it.
I really appreciate your help. Kindly guide me what could be the reason?
